src/config/SPRACINGH7NANO/config.h

```diff
@@ -0,0 +1,199 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32H750
+
+#define BOARD_NAME        SPRACINGH7NANO
+#define MANUFACTURER_ID   SPRO
+
+#define TARGET_BOARD_IDENTIFIER "SP7N"
+#define USBD_PRODUCT_STRING "SPRacingH7NANO"
+
+#define FC_VMA_ADDRESS    0x97CE0000
+
+#define EEPROM_SIZE 8192
+
+#define USE_SPRACING_PERSISTENT_RTC_WORKAROUND
+
+#define USE_BUTTONS
+#define BUTTON_A_PIN            PE4
+#define BUTTON_A_PIN_INVERTED
+#define BUTTON_B_PIN            PE4
+#define BUTTON_B_PIN_INVERTED
+
+#define USE_QUADSPI
+#define USE_QUADSPI_DEVICE_1
+#define QUADSPI1_SCK_PIN PB2
+#define QUADSPI1_BK1_IO0_PIN PD11
+#define QUADSPI1_BK1_IO1_PIN PD12
+#define QUADSPI1_BK1_IO2_PIN PE2
+#define QUADSPI1_BK1_IO3_PIN PD13
+#define QUADSPI1_BK1_CS_PIN NONE
+#define QUADSPI1_BK2_IO0_PIN PE7
+#define QUADSPI1_BK2_IO1_PIN PE8
+#define QUADSPI1_BK2_IO2_PIN PE9
+#define QUADSPI1_BK2_IO3_PIN PE10
+#define QUADSPI1_BK2_CS_PIN PB10
+#define QUADSPI1_MODE QUADSPI_MODE_BK2_ONLY
+#define QUADSPI1_CS_FLAGS (QUADSPI_BK1_CS_NONE | QUADSPI_BK2_CS_SOFTWARE | QUADSPI_CS_MODE_SEPARATE)
+
+#define FLASH_QUADSPI_INSTANCE    QUADSPI
+
+#define CONFIG_IN_EXTERNAL_FLASH
+
+// SD card not present on hardware, but pins are reserved.
+//#define USE_SDCARD
+#ifdef USE_SDCARD
+#define SDCARD_DETECT_PIN PD10
+#define SDCARD_DETECT_INVERTED
+#define DEFAULT_BLACKBOX_DEVICE     BLACKBOX_DEVICE_SDCARD
+#else
+#define DEFAULT_BLACKBOX_DEVICE     BLACKBOX_DEVICE_FLASH
+#endif
+
+#define USE_SPI
+
+#define USE_SPI_DEVICE_2
```

Comment:
Both already defined in src/main/target/STM32H750/target.h
Same for line 82 and 88